docs/docs/developer-guides/oracle-set-up-guide.md

```diff
@@ -6,17 +6,19 @@
 [Apollo](../products/apollo-overview.md) is an oracle for submitting prices on-chain. 
 We plan to upgrade it and allow everyone to stream arbitrary third party data in the future. 
 By taking part in Apollo and becoming an oracle operator, 
-you will be securing the assets prices and help the network become more resilient. 
+you will be securing the price of assets on-chain and help the network become more resilient in a decentralized manner. 
 Participants receive tokens as rewards for their work, 
 but it is important to note that the on-chain algorithm also allows for slashing based on bad behavior, 
 which is defined as submitting prices out of a certain threshold range from a pivot price calculated on all submitted prices.
 
-Apollo consists of three key components. The On-chain pallet/worker that decides when a price is requested. 
-An Off-Chain worker that monitors when a price has been requested and if so, submits a price for the request. 
-Finally, a Price-Feed (we provide a reference implementation for this component) that fetches the prices from a CEX/DEX 
-and caches them, such that the Off-Chain worker is able to query those prices and stream them by submitting a transaction. 
-Below is a high level diagram that shows the interactions between the components.
+Apollo consists of three key components: 
 
+- The On-chain pallet/worker that decides when a price is requested. 
```

Comment:
remove worker

Reply:
worker is mentioned many times, should I replace it everywhere?